plan/testdata/ctas_basic.json

```diff
@@ -0,0 +1,143 @@
+                        "department_id",
+                        "salary",
+                        "role"
+                      ],
+                      "struct": {
+                        "types": [
+                          {
+                            "i32": {
+                              "nullability": "NULLABILITY_REQUIRED"
+                            }
+                          },
+                          {
+                            "string": {
+                              "nullability": "NULLABILITY_NULLABLE"
+                            }
+                          },
+                          {
+                            "i32": {
+                              "nullability": "NULLABILITY_NULLABLE"
+                            }
+                          },
+                          {
+                            "decimal": {
+                              "scale": 2,
+                              "precision": 10,
+                              "nullability": "NULLABILITY_NULLABLE"
+                            }
+                          },
+                          {
+                            "string": {
+                              "nullability": "NULLABILITY_NULLABLE"
+                            }
+                          }
+                        ],
+                        "nullability": "NULLABILITY_REQUIRED"
+                      }
+                    },
+                    "projection": {
+                      "select": {
+                        "structItems": [
+                          {
+                            "field": 1
+                          },
+                          {
+                            "field": 3
+                          }
+                        ]
+                      },
+                      "maintainSingularStruct": true
+                    },
+                    "namedTable": {
+                      "names": [
+                        "employees"
+                      ]
+                    }
+                  }
+                },
+                "expressions": [
+                  {
+                    "selection": {
+                      "directReference": {
+                        "structField": {
+                          "field": 0
+                        }
+                      },
+                      "rootReference": {}
+                    }
+                  },
+                  {
+                    "selection": {
+                      "directReference": {
+                        "structField": {
+                          "field": 1
+                        }
+                      },
+                      "rootReference": {}
+                    }
+                  }
+                ]
+              }
+            }
+          }
+        },
+        "names": [
+          "name",
+          "salary"
+        ]
+      }
+    }
+  ],
+  "version": {
+    "majorNumber": 0,
+    "minorNumber": 29,
```

Comment:
Optional:  Every time we bump the protocol version we'll need to update this file although it doesn't necessarily affect anything.  That may be fine as it may prompt us to double check this file's construction but I suspect it'll be a nuisance.  Does checkRoundTrip depend on this?

Reply:
@EpsilonPrime Yes, checkRoundTrip depends on this.